hub/config/src/integration-test/java/uk/gov/ida/integrationtest/hub/config/apprule/support/ConfigAppRule.java

```diff
@@ -77,7 +77,7 @@ public ConfigAppRule(ConfigOverride... configOverrides) {
                 .add(config("translationsDirectory", TRANSLATIONS_RELATIVE_PATH))
                 .add(configOverrides)
                 .build();
-        return overrides.toArray(new ConfigOverride[overrides.size()]);
+        return overrides.toArray(new ConfigOverride[0]);
```

Comment:
Why not initialise the array to be the correct size. Otherwise it ends up creating a new one of the correct size anyway.

Reply:
https://shipilev.net/blog/2016/arrays-wisdom-ancients/

Reply:
Isn't that interesting.